### ISSUE TITLE
Fix false positives for font names containing keywords in value-keyword-case

### ISSUE THREAD

**Issue by thewilkybarkid** [stylelint/stylelint#5384]
> Clearly describe the bug

Font names that contain a keyword (eg 'Source Serif') are case checked when they shouldn't be.

> Which rule, if any, is the bug related to?

`value-keyword-case`

> What code is needed to reproduce the bug?

```css
body {
  font-family: Source Serif;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`13.13.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
Expected "Serif" to be "serif"   value-keyword-case
```

**Comment by jeddy3** [comment]
@thewilkybarkid Thanks for the report and for using the template.

The [spec](https://www.w3.org/TR/CSS2/fonts.html#font-family-prop) recommends quoting font family names containing spaces, and you can enforce that using the [`font-family-name-quotes` rule](https://stylelint.io/user-guide/rules/list/font-family-name-quotes/). Having said that, I believe your example is valid CSS and so we should fix the `value-keyword-case` rule with a special case for the `font` and `font-family` properties.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by thewilkybarkid** [comment]
👍 

I tried using `always-where-recommended`, but it conflicts by wanting to remove the quotes.

**Comment by jeddy3** [comment]
> I tried using `always-where-recommended`, but it conflicts by wanting to remove the quotes.

Please can you provide some more detail, as I can't replicate it: 

[Demo without quotes showing violations](https://stylelint.io/demo/#N4Igxg9gJgpiBcIBG0CeACYAdAdu9AZhDgC4C0BAhgLYCWANqvOgMoQCuATmDKzJ7QIBuXAF8QAGhABnVKUoAPBOGnTJ4YgVoBzZdjzosITu3oxpR5vvz4jRUhRoNUZHDRhkAjuwglzlwxBKegB3SlRpMhCAC34PThhIamoYHFgoIwlcG0CAN2D2DwBrGFQQiE4oMjBKaRgAo3oIEP4jbPRRMRBRIA)  and [demo with quotes has no violations](https://stylelint.io/demo/#N4Igxg9gJgpiBcIBG0CeACYAdAdu9AZhDgC4C0BAhgLYCWANqvOliAMoQCuATmDOmxjdaBVgG5cAXxAAaEAGdUpSgA8E4efNnhiBWgHN12PCxDdO9GPNbNj+fKyKkKNBqjI4aMMgEdOEEisbU0p6AHdKVHkyMIALIW9uGEhqahgcWChWGVx7UwA3UM5vAGsYVDCIbigyMEp5GGDWeggwoVZc9EkpEEkgA).

**Comment by thewilkybarkid** [comment]
Sorry, meant `always-where-required`: https://stylelint.io/demo/#N4Igxg9gJgpiBcIBG0CeACYAdAdu9AZhDgC4C0BAhgLYCWANqvOliAMoQCuATmDOmxjdaBVgBp0AZyEiA3LgC+IMSEmpSlAB4JwkycvDECtAOY7seFiG6d6MSa2YX8+VkVIUaDVGRw0YZACOnBAk9o5WlPQA7pSokmTRABZCAdwwwbTpUOK4LlYAblGcAQDWMKjRENxQZGCU0hGs9BDRQqx56AqKIApAA

**Comment by jeddy3** [comment]
That's the intended behaviour. Quotes aren't _required_ around family names containing spaces, but they are _recommended_.  The [options sections](https://stylelint.io/user-guide/rules/list/font-family-name-quotes/#options) in the doc goes into more details about this.

I recommend using the `always-where-recommended` option.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
